### PR TITLE
fix(core): tweak swipes in ETH staking flow

### LINKED ISSUE #4167: Not possible to swipe back in /// section when confirming Staking or sending Eth
**Describe the bug**
It's not possible to swipe back from info screen during Eth transaction signing.
Bitcoin works as expected

**Firmware version and revision**
**Info:**
 - Suite version: web 24.10.0 (2c19c7365c947ed2e1e4dbfeda6e5bfb4353f33f)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.2 regular (revision 37006592dbc3f2d817def0c75ed6a62513895ec5)
 - Transport: WebUsbTransport

**To Reproduce**
1. Initialize tx Stake transaction
2. in Section `Stake ETH on Everstake` open /// section
3. Try to swipe back
4. Open `Stake address` 
5. try to sipe back

**Expected behavior**
You were able to swipe back from /// with 2.8.0 firmware

### PR DESCRIPTION
**Pull request by ibz**
Fixes #4167

### CODE REVIEW

File: core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs [hunk 1]
```diff
@@ -260,6 +263,7 @@ fn new_confirm_output_obj(_args: &[Obj], kwargs: &Map) -> Result<Obj, error::Err
         .with_footer(TR::instructions__swipe_up.into(), None)
         .with_chunkify(chunkify)
         .with_text_mono(text_mono)
+        .with_swipe_up()
```
Comment (obrusvit) [inline core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs:266]: This was missing? :thinking:

File: core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs [hunk 2]
```diff
@@ -307,6 +311,7 @@ fn new_confirm_output_obj(_args: &[Obj], kwargs: &Map) -> Result<Obj, error::Err
                 .with_menu_button()
                 .with_footer(TR::instructions__swipe_up.into(), None)
                 .with_text_mono(text_mono)
+                .with_swipe_up()
```
Comment (obrusvit) [inline core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs:314]: ditto

**Review by obrusvit — APPROVED** [review]
utACK

File: core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs (hunk 1 above)
Comment (ibz) [reply core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs:266]: It defaulted to `true` before, but I made it default to `false` so I had to add it explicitly to the places where it was actually needed.

It is actually unrelated to the issue at hand, but I noticed that if this is `true` on the screen where address is confirmed, you can swipe up and the address gets our of the screen and you are stuck with a strange empty screen... Which is why I made it `false` in that case.

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 24.11.0 (850d7ee26ec9c8274058f60bbaba0cc4c29de357)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.11.0 Chrome/128.0.6613.162 Electron/32.1.2 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.4 regular (revision d71d9e9c343ad04a7f28d6b02f8d3f8945af30d6)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**